Classes/common/CDTIncrementalStore.m

```diff
@@ -16,6 +16,11 @@
 #import "CDTFieldIndexer.h"
 #import "CDTISGraphviz.h"
 
+@implementation NSBatchUpdateResult
+- (void)setResultType:(NSBatchUpdateRequestResultType)resultType { _resultType = resultType; }
+- (void)setResult:(id)result { _result = result; }
+@end
```

Comment:
Wow, that's all you needed to do?
It goes against my understanding, but I'll take it.